app/views/project_summary_mailer/project_summary.html.slim

```diff
@@ -0,0 +1,35 @@
+h3
+  = "Dear, #{@user.full_name}"
+p
+  | Here is a summary of the changed in your board
+p
+  - unless @user.new_tasks.empty?
+    |NEW TASKS 
+    - @user.new_tasks.each do |newtask|
+        = newtask.title
```

Comment:
not sure what would look better so you can either add br after line 9 (in loop) or

```ruby
 - unless @user.new_tasks.empty?
     p NEW TASKS 
     - @user.new_tasks.each do |newtask|
     p = newtask.title
```